### PR TITLE
Implement cameras launch file with synchronisation

### PR DESCRIPTION
**Pull request by GreatAlexander**
- Use ROS2 spinnaker synchronized camera driver instead of default
 - Modify camera config parameter yaml files to fit new structure
 - Add exposure controller parameters as required
 - Update all_cams.launch.xml to run all cameras in single node
 - NOTE: Change exposure target to 1ms
 - NOTE: Also compute brightness in /meta as it may be useful

### CODE REVIEW

File: av_camera_launch/config/fsp_l_config.yaml [hunk 1]
```diff
@@ -1,36 +1,47 @@
...
+      exposure_controller:
+        brightness_target: 120
+        brightness_tolerance: 20
+        gain_priority: false
+        max_exposure_time: 1000
+        max_frames_skip: 10
+        max_gain: 10.0
+        min_exposure_time: 1000
+        type: "master"
+      exposure_time: 5000
```
Comment (hect95) [inline av_camera_launch/config/fsp_l_config.yaml:26]: I think we want this at `1ms` ? 
```suggestion
      exposure_time: 1000
```

File: av_camera_launch/launch/all_cams.launch.xml [hunk 2]
```diff
@@ -1,14 +1,20 @@
 <launch>
   <group>
-    <push-ros-namespace namespace="sensor/camera"/>
+    <push-ros-namespace namespace="/sensor/camera"/>
     <node_container pkg="rclcpp_components" exec="component_container_mt" name="camera_container" namespace="">
-      <!-- fsp_l -->
-
-      <composable_node pkg="spinnaker_camera_driver" plugin="spinnaker_camera_driver::CameraDriver" name="fsp_l" namespace="">
+      <!-- Synchronised Camera Driver composable node -->
+      <composable_node pkg="spinnaker_synchronized_camera_driver" plugin="spinnaker_synchronized_camera_driver::SynchronizedCameraDriver" name="camera" namespace="/sensor">
```
Comment (hect95) [inline av_camera_launch/launch/all_cams.launch.xml:6]: Wouldn't the composable node grasp the `namespace` defined above? 

Can we call this node `spinnaker_camera_sync_node` ? :)

**Review by hect95 — CHANGES_REQUESTED** [review]

File: av_camera_launch/config/fsp_l_config.yaml (hunk 1 above)
Comment (hect95) [reply av_camera_launch/config/fsp_l_config.yaml]: Fixed hehe

**Comment by GreatAlexander** [comment]
NOTE:
 - We need to enable compute brightness to use the exposure controller as part of the synchronised driver
 - Unfortunately, BayerBG8 (inverse images) is not supported for computing brightness. So temporarily until it is fixed upstream we will revert to `gain_auto` "Continuous" and `compute_brightness` "False"

**Pull request merged**